packages/compass-e2e-tests/tests/collection-bulk-delete.test.ts

```diff
@@ -0,0 +1,117 @@
+import { expect } from 'chai';
+import type { CompassBrowser } from '../helpers/compass-browser';
+import { startTelemetryServer } from '../helpers/telemetry';
+import type { Telemetry } from '../helpers/telemetry';
+import { beforeTests, afterTests, afterTest } from '../helpers/compass';
+import type { Compass } from '../helpers/compass';
+import * as Selectors from '../helpers/selectors';
+import { createNumbersCollection } from '../helpers/insert-data';
+
+describe('Bulk Delete', () => {
+  let compass: Compass;
+  let browser: CompassBrowser;
+  let telemetry: Telemetry;
+
+  before(async function () {
+    telemetry = await startTelemetryServer();
+    compass = await beforeTests({
+      extraSpawnArgs: ['--enableBulkDeleteOperations'],
+    });
+    browser = compass.browser;
+  });
+
+  after(async function () {
+    await afterTests(compass, this.currentTest);
+    await telemetry.stop();
+  });
+
+  beforeEach(async function () {
+    await createNumbersCollection();
+    await browser.connectWithConnectionString();
+    await browser.navigateToCollectionTab('test', 'numbers', 'Documents');
+  });
+
+  afterEach(async function () {
+    await afterTest(compass, this.currentTest);
+  });
+
+  it('deletes documents matching a filter', async function () {
+    //const telemetryEntry = await browser.listenForTelemetryEvents(telemetry);
+
+    // Set a query that we'll use.
+    await browser.runFindOperation('Documents', '{ i: 5 }');
+
+    // Open the modal.
+    await browser.clickVisible(Selectors.OpenBulkDeleteButton);
+    await browser.$(Selectors.BulkDeleteModal).waitForDisplayed();
+
+    // TODO(COMPASS-7448): Make sure the query is shown in the modal.
+
+    // Check that it will update the expected number of documents
+    expect(await browser.$(Selectors.BulkDeleteModalTitle).getText()).to.equal(
+      'Delete 1 document'
+    );
+
+    expect(
+      await browser.$(Selectors.BulkDeleteModalDeleteButton).getText()
+    ).to.equal('Delete 1 document');
+
+    // Press delete
+    await browser.clickVisible(Selectors.BulkDeleteModalDeleteButton);
+
+    // Press delete in the confirmation modal
+    await browser.clickVisible(Selectors.ConfirmationModalConfirmButton());
+
+    // The modal should go away
```

Comment:
Nitpick: probably before the confirmation modal step?

Reply:
Yeah sounds good, let me do the change.